src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseController.java

```diff
@@ -111,6 +117,15 @@ public CaseData updateCase(
             );
             CaseData updatedCaseData = CaseUtils.getCaseData(caseDetails, objectMapper);
             updatedCaseData = confidentialDetailsMapper.mapConfidentialData(updatedCaseData, true);
+            if (CITIZEN_CASE_SUBMIT.getValue().equalsIgnoreCase(eventId)
+                || CITIZEN_CASE_SUBMIT_WITH_HWF.getValue().equalsIgnoreCase(eventId)) {
+                updatedCaseData = reasonableAdjustmentsMapper.mapRAforC100MainApplicant(
+                    caseData.getC100RebuildData().getC100RebuildApplicantDetails(),
+                    updatedCaseData,
+                    authorisation
+                );
+            }
```

Comment:
not sure if this is the right place?

Reply:
Moved the check inside mapRAforC100MainApplicant method.